app/models/contributor.rb

```diff
@@ -32,12 +33,14 @@ class Contributor < ApplicationRecord
   validates :data_processing_consent, acceptance: true, unless: proc { |c| c.editor_guarantees_data_consent }
 
   validates :email, uniqueness: { case_sensitive: false }, allow_nil: true, 'valid_email_2/email': true
-  validates :threema_id, uniqueness: { case_sensitive: false }, allow_blank: true, format: { with: /\A[A-Za-z0-9]+\z/ }, length: { is: 8 }
+  validates :threema_id, uniqueness: { case_sensitive: false }, allow_blank: true
```

Comment:
we still want to validate that we don't have any contributors already with the threema_id, ignoring case